pathfinding_service.rst

```diff
@@ -298,23 +298,9 @@ Fields
 +--------------------------+------------+--------------------------------------------------------------------------------+
 | Field Name               | Field Type |  Description                                                                   |
 +==========================+============+================================================================================+
-| chain_id                 | uint256    | Chain identifier as defined in EIP155                                          |
+| balance_proof            | object     | `Offchain Balance Proof`__                                                     |
 +--------------------------+------------+--------------------------------------------------------------------------------+
-| token_network_identifier | address    | Address of the TokenNetwork contract                                           |
-+--------------------------+------------+--------------------------------------------------------------------------------+
-| channel_identifier       | uint256    | Channel identifier inside the TokenNetwork contract                            |
-+--------------------------+------------+--------------------------------------------------------------------------------+
-| updating_participant     | address    | Channel participant who sends the balance update                               |
-+--------------------------+------------+--------------------------------------------------------------------------------+
-| other_participant        | address    | Channel participant who doesn't send the balance update                        |
-+--------------------------+------------+--------------------------------------------------------------------------------+
-| updating_nonce           | uint256    | Strictly monotonic value used to order transfers. The nonce starts at 1        |
-+--------------------------+------------+--------------------------------------------------------------------------------+
-| other_nonce              | uint256    | Strictly monotonic value used to order transfers. The nonce starts at 1        |
-+--------------------------+------------+--------------------------------------------------------------------------------+
-| updating_capacity        | uint256    | Available capacity for the participant sending the update                      |
-+--------------------------+------------+--------------------------------------------------------------------------------+
-| other_capacity           | uint256    | Available capacity for the participant not sending the update                  |
+| our_nonce                | uint256    | Address of the TokenNetwork contract                                           |
```

Comment:
I guess you meant `Nonce of the latest Offchain Balance Proof that the sender of the message has sent to the sender of 'balance_proof'`?